include/fpm/fixed.hpp

```diff
@@ -487,4 +487,17 @@ constexpr bool numeric_limits<fpm::fixed<B,I,F,R>>::tinyness_before;
 
 }
 
+namespace fpm
+{
+template<typename T>
+struct is_fixed : std::false_type {};
+
+template<typename BaseType, typename IntermediateType, unsigned int FractionBits, bool EnableRounding>
+struct is_fixed<fixed<BaseType, IntermediateType, FractionBits, EnableRounding>> : std::true_type {};
+
+#ifdef __cpp_lib_bool_constant
```

Comment:
The build falls over the `inline` variable below, which isn't allowed until C++17. I think you should check `__cplusplus` in this ifdef.

Reply:
I thought I've used the correct "feature" `#ifdef` (I have little experience with versioning like this). I'll change it to the `__cplusplus` ~~tomorrow~~ soon.